src/main/java/fr/cnes/sonarqube/plugins/framac/sensor/FramaCSensor.java

```diff
@@ -61,6 +67,27 @@ public void describe(final SensorDescriptor sensorDescriptor) {
         sensorDescriptor.createIssuesForRuleRepositories(FramaCRulesDefinition.getRepositoryKeyForLanguage());
     }
 
+    /**
+     * Check if a file is a CSV file by analysing the header (first line)
+     * @param filePath : the file path
+     * @return true if the file is a csv file, false otherwise
+     */
+    private boolean isCsvFile(String filePath) {
+        boolean isCsvFile = false;
+        try (BufferedReader br =
+            new BufferedReader(new FileReader(filePath))) {
+            String line = br.readLine();
+            if (line.startsWith("directory\tfile")) {
+                isCsvFile = true;
+            }
+        } catch (IOException e) {
+            //do nothing, if problem, error will be raised by analyse function}
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/lequal/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fr.cnes.sonarqube.plugins%3Asonar-frama-c-plugin&pullRequest=23&issues=AWY0CuRBvOWp7Yd6eg9p&open=AWY0CuRBvOWp7Yd6eg9p)

[ISSUE_KEY]: AWY0CuRBvOWp7Yd6eg9p